docs/DeveloperGuide.md

```diff
@@ -407,6 +407,12 @@ Given below is an example scenario to reset the timetable.
 |v1.0|user|add modules to the application|keep track of them during the school term|
 |v1.0|user|delete tasks from the application|keep track of new tasks|
 |v1.0|user|delete modules from the application|keep track of new modules|
+|v1.0|user|edit task descriptions|make changes or updates to my tasks|
+|v1.0|user|edit module code|make changes to module codes in the list|
+|v1.0|user|mark task as done|update the completion status of my tasks|
+|v1.0|user|view my task list|be aware of the tasks I have added to my list|
+|v1.0|user|view my module list|be aware of the modules I have added to my list|
+|v2.0|user|receive help messages and prompts|seamlessly navigate and use ra.VI's features|
```

Comment:
Not too sure about this user story.\
Maybe something like receive help easily so that I know how to use ravi efficiently